Makefile.am

```diff
@@ -308,6 +310,9 @@ endef
 install-directories-hook:
 	$(MKDIR_P) $(addprefix $(DESTDIR),$(INSTALL_DIRS))
 
+install-environment-conf-hook:
+	$(AM_V_LN)$(LN_S) -f $(sysconfdir)/environment $(DESTDIR)$(environmentconfdir)/99-environment.conf
```

Comment:
I think it would be best if it were relative. This way OS trees do not have symlinks pointing outside their base dir. `$(AM_V_LN)$(LN_S) --relative -f $(DESTDIR)$(sysconfdir)/environment $(DESTDIR)$(environmentconfdir)/99-environment.conf`